docker-compose.yml

```diff
@@ -0,0 +1,56 @@
+version: '3'
+services:
+  zookeeper:
+    image: confluentinc/cp-zookeeper:5.1.0
+    hostname: zookeeper
+    container_name: zookeeper
+    expose:
+      - "2181"
```

Comment:
To be able to run tests as I was mentioning just above we just need to forward a few ports here, as long as they are not default ports we should be fine and not at risk of colliding with the host kafka / zookeeper/etc